index.html

```diff
@@ -0,0 +1,131 @@
+            <ul class="stack-three">
+                <li><h3>Skill</h3></li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </section> 
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="social">
+            <div class="social-media">
+            <ul class="social-media-icon">
+                <li><a href="#"><img src="./assets/icon/github.png" alt="Github"></a></li>
+                <li><a href="#"><img src="./assets/icon/linkedin.png" alt="Linkedin"></a></li>
+                <li><a href="#"><img src="./assets/icon/Vector.png" alt="Angellist"></a></li>
+                <li><a href="#"><img src="./assets/icon/twitter.png" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/icon/medium.png" alt="Medium"></a></li>
+            </ul>
+                <a class="arrow" href="#"><img src="./assets/icon/element-right.png" alt="arrow"></a>   
+            </div>                      
+        </section>       
+        <div class="experience-title">
+            <h2>MY RECENT <br> WORK</h2>
+        </div>        
+        <Span class="divider"><img src="./assets/images/line-diverder.png" alt="section divider"></Span>
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li class="border-right padding-right">css</li>
+                <li class="border-right padding-right">html</li>          
+                <li class="border-right padding-right">bootstrap</li>         
+                <li class="padding-right">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+        </section> 
+        <section class="experience-container">
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li class="border-right padding-right">css</li>
+                <li class="border-right padding-right">html</li>          
+                <li class="border-right padding-right">bootstrap</li>         
+                <li class="padding-right">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
```

Comment:
> * [ ]  As requested in the previous review, please wrap the entire project section in a `ul` tag and each project item should be a `li` tag. So rather than `section` use `li` tag and then wrap all `li` tags in a `ul` tag.

- [ ] Kindly fix the above-mentioned issue.

Reply:
hello @mikenath233,
I have added the grid in the ul as you asked unless I just didn`t understand what your asking. My mentor is never in touch and my partners sleeping now so I would really appreciate your help in making this clear for me.

Reply:
- Something like this:
```
<ul class='projects-container'>
  <li class='experience-container'>project-one</li>
  <li class='experience-container'>project-two</li>  
  <li class='experience-container'>project-three</li>
  ...
</ul>
```